src/app/services/website.service.ts

```diff
@@ -91,8 +91,8 @@ export class WebsiteService extends CRUDListService<ApiWebsiteWithCompany, ApiWe
 
 
   readonly extractUnregistered = (q?: string, start?: Date, end?: Date): Observable<HostWithReportCount[]> => {
-    return this.utils.getSecuredReportApiSdk.pipe(
-      mergeMap(api => api.website.extractUnregistered(
+    return of(_ => _.next()).pipe(
```

Comment:
à quoi sert `of(_ => _.next())` ? équivalent à `empty()` ?

Reply:
Oui c'est ça :). 
Du coup c'est `EMPTY`, `empty()` est déprécié.